modules/ztp-image-based-upgrade-installing-lifecycle-agent-using-cli.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-image-based-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-lcao-using-cli_{context}"]
+= Installing the {lcao} by using the CLI
+
+You can use the OpenShift CLI (`oc`) to install the {lcao}.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Install the latest version of the {rh-rhacm} Operator.
+* Set up a hub cluster with disconnected registry.
```

Comment:
If disconnected registry is indeed a prereq, what does this mean for the quay.io container image references you have dotted elsewhere?

Reply:
I'll have to check this because I used the generic installation procedure here.